impl/src/main/java/com/sun/faces/application/WebappLifecycleListener.java

```diff
@@ -18,14 +18,16 @@
 
 import static com.sun.faces.application.view.ViewScopeManager.ACTIVE_VIEW_MAPS;
 import static com.sun.faces.application.view.ViewScopeManager.VIEW_SCOPE_MANAGER;
+import static com.sun.faces.cdi.clientwindow.ClientWindowScopeManager.CLIENT_WINDOW_SCOPE_MANAGER;
 import static com.sun.faces.config.WebConfiguration.BooleanWebContextInitParameter.EnableDistributable;
 
+import static java.util.Arrays.asList;
```

Comment:
I don't see where you are using this static import?

Reply:
https://github.com/eclipse-ee4j/mojarra/pull/4921/files#diff-190982bf21163fdfde1ddb8c06b6d70f4f57e608fd44c894959c1c45f4db56dfR148